### DISCUSSION TITLE
能否把“倍速记忆”与它的子功能“扩展倍速菜单”分开来

### DISCUSSION THREAD

**Discussion by SHIRO-YOSHONO** [the1812/Bilibili-Evolved#3125] [功能建议 / Ideas]
RT，我希望单独使用“扩展倍速菜单”，倍速记忆有时候会让我正常看视频有点困扰，望采纳

**Comment by Geeyun-JY3** [comment]
请
善
用
搜
索
相关 PR：#2746
相关 issue、讨论：#2478、#2623、#2660、#2723、#2744
